src/modals/MappingTest/components/TestMappingResult.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+	<div>
+		<h4>Test result</h4>
+
+		<div class="content">
+			<NcNoteCard v-if="mappingTest.success" type="success">
+				<p>Mapping successfully tested</p>
+			</NcNoteCard>
+			<NcNoteCard v-if="mappingTest.success === false" type="error">
+				<p>Mapping failed to test</p>
+			</NcNoteCard>
+			<NcNoteCard v-if="mappingTest.error" type="error">
+				<p>{{ mappingTest.error }}</p>
+			</NcNoteCard>
+
+			<NcGuestContent class="result">
```

Comment:
The component `<NcGuestContent />` should only be used as a main wrapper on a page.
And because it isn't used as such here it causes console errors